tests/cli/test_cli.py

```diff
@@ -25,17 +25,55 @@ def test_helps_no_raise(self):
             "optimum-cli export --help",
             "optimum-cli export onnx --help",
             "optimum-cli env --help",
+            "optimum-cli onnxruntime quantize --help",
+            "optimum-cli onnxruntime optimize --help",
         ]
 
         for command in commands:
             subprocess.run(command, shell=True, check=True)
 
-    def test_basic_commands(self):
+    def test_env_commands(self):
+        subprocess.run("optimum-cli env", shell=True, check=True)
+
+    def test_export_commands(self):
         with tempfile.TemporaryDirectory() as tempdir:
-            commands = [
-                "optimum-cli env",
+            command = (
                 f"optimum-cli export onnx --model hf-internal-testing/tiny-random-vision_perceiver_conv --task image-classification {tempdir}",
+            )
+            subprocess.run(command, shell=True, check=True)
+
+    def test_optimize_commands(self):
+        with tempfile.TemporaryDirectory() as tempdir:
+            # First export a tiny encoder, decoder only and encoder-decoder
+            export_commands = [
+                f"optimum-cli export onnx --model hf-internal-testing/tiny-random-BertModel {tempdir}/encoder",
+                f"optimum-cli export onnx --model hf-internal-testing/tiny-random-gpt2 {tempdir}/decoder",
+                f"optimum-cli export onnx --model hf-internal-testing/tiny-random-t5 {tempdir}/encoder-decoder",
+            ]
+            optimize_commands = [
+                f"optimum-cli onnxruntime optimize --onnx_model {tempdir}/encoder -O1",
+                f"optimum-cli onnxruntime optimize --onnx_model {tempdir}/decoder -O1",
+                f"optimum-cli onnxruntime optimize --onnx_model {tempdir}/encoder-decoder -O1",
```

Comment:
Maybe add test with custom config here

Reply:
Do you already have an ORTConfig file for testing usage somewhere?

Reply:
@fxmarty @michaelbenayoun I have an issue for testing the `ORTConfig` parameter. Apparently the parameters `ORTConfig.optimization` and `ORTConfig.quantization` are dictionaries. How can I get them back to usual dataclass object? As the `quantize` and `optimize` methods expect to have objects and not dictionaries.

Reply:
I could not find one so forget about i!